components/Template.js

```diff
@@ -67,7 +67,13 @@ function transformData(fn, templateKey, originalData) {
     data = fn(originalData);
   } else if (typeof fn === 'object') {
     // ex: transformData: {hit, empty}
-    data = fn[templateKey] && fn[templateKey](originalData);
```

Comment:
`data = fn[templateKey] && fn[templateKey](originalData) || originalData`, no? :) Even if it does what we want it would less explicit ;)